e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @profile_popover
+
+describe('Profile popover', () => {
+    let testTeam;
+    let testUser;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+        });
+    });
+
+    it('MM-T2 Add user — Error if already in channel', () => {
+        // Login as test user and go to town square
+        cy.apiLogin(testUser);
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // Send multiple messages so that the profile popover appears completely.
+        cy.postMessage('Hi there\nsending\na\nmessage');
+        cy.apiLogout();
+
+        // Login as sysadmin now
+        cy.apiAdminLogin();
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // Open profile popover
+        cy.get('#postListContent').within(() => {
+            cy.findAllByText(testUser.username).first().should('have.text', testUser.username).click();
+        });
+
+        // Add to a Channel
+        cy.findByText('Add to a Channel').click();
+
+        cy.get('div[aria-labelledby="addChannelModalLabel"]').within(() => {
+            // Type "Town"
+            cy.get('input').should('be.visible').type('Town');
```

Comment:
Instead of clicking the dropdown, I wanted to do `type('Town{enter}')` to simulate pressing enter. But it doesn't seem to work. Any idea why?

Reply:
Probably it needs some time to find the channel. 
```suggestion
            cy.get('input').should('be.visible').type('Town').wait(TIMEOUTS.HALF_SEC).type('{enter}');
```

Reply:
Amazing. That worked! :bow: